src/provider/proxy/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const COLLECTION_PATHS = {
```

Comment:
dupplicate values with file [packages/webrtc-firebase/src/constants.ts](https://github.com/saigontechnology/rn-video-call/pull/6/files#diff-9d36b1c76d2cd1798a531b5692e99b2c9eafd5e74ba4fcb4c510d288ff294558)

Reply:
just a temporary, we will remove when packages repo is ready.